pkg/cmd/create/cluster/scenario/create/hub/openstack/install_config.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+metadata:
+  name: "{{ .managedCluster.name }}"
+baseDomain: {{ .managedCluster.openstack.baseDnsDomain }}
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  replicas: {{ if .managedCluster.master }}{{ .managedCluster.master.replicas }}{{ else }}3{{end}}
+  platform:
+    openstack:
+      type: {{ .managedCluster.openstack.masterFlavor }}
```

Comment:
Can you put a default value, like it is done for the replicas?

Reply:
I had a feeling you might ask for this...

Reply:
Something like 

```
  replicas: {{ if .managedCluster.master }}{{ .managedCluster.master.replicas }}{{ else }}3{{end}}
  platform:
    openstack:
      type: {{ default .managedCluster.openstack.masterFlavor default_flavor }}
```

Reply:
I used an if else like in the other case